src/Smalot/PdfParser/Font.php

```diff
@@ -468,6 +475,12 @@ public function decodeText(array $commands): string
      */
     public function decodeContent(string $text, bool &$unicode = null): string
     {
+        // If this string begins with a UTF-16BE BOM, then decode it
+        // directly as Unicode
+        if (preg_match('/^\xFE\xFF/i', $text)) {
```

Comment:
Just a question: Why not use `substr` to get first two characters and check them instead? Using `preg_match` might be more "expensive".

Reply:
I used `preg_match()` only because the existing `decodeUnicode()` already uses the same check. I can certainly change both. :)

Reply:
Only change if you see fit. It is up to you.